code-challenges.test.js

```diff
@@ -25,7 +28,26 @@ const colors2 = [
   "aquamarine",
   "saffron"
 ]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// Psedocode 1. Create a function called firstItem with array as a parameter
+//           2. Input is a function that takes in an array 
+//           3. Use .shift to remove the first element from an array 
+//           4. Use .sort to return array in order with math.random to shuffle the order.
```

Comment:
Nice! I really like this approach and pseudocode. Just a few things to consider, does shift need "" and is our sort working as intended? Meaning, is it shuffling the remaining elements in the array?